docs/capacity-planning.rst

```diff
@@ -0,0 +1,326 @@
+.. _ksql_capacity_planning:
+
+======================
+KSQL Capacity Planning
+======================
+
+.. contents::
+
+KSQL is a simple and powerful tool for building streaming applications on top of Kafka. This guide will help you get started with planning for provisioning your KSQL deployment by helping you answer questions like:
+
```

Comment:
the bullets in lines 11-15 shouldn't be indented, otherwise the contents are shown like this (with an extra vertical bar):
![image](https://user-images.githubusercontent.com/11722533/39157810-6fbfef62-4711-11e8-972f-863ada09f50d.png)